lib/grpc/message.ex

```diff
@@ -47,7 +48,13 @@ defmodule GRPC.Message do
       {:error, "Encoded message is too large (#{length} bytes)"}
     else
       result = <<compress_flag, length::size(4)-unit(8), message::binary>>
-      {:ok, result, length + 5}
+
+      data =
+        if not is_nil(codec) and function_exported?(codec, :pack_for_channel, 1),
```

Comment:
```suggestion
        if function_exported?(codec, :pack_for_channel, 1),
```

We might have to deal with `Code.ensure_loaded!` here, but I think it's safe to do it just like this for now